config/routes.rb

```diff
@@ -3,7 +3,8 @@
   mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
   devise_for :users
   root 'static_pages#home'
-  get 'kontakt' => 'static_pages#kontakt'
+  match '/contact' , to: 'static_pages#contact', via: 'get'
+  match '/contact' , to: 'static_pages#mail', via: 'post'
```

Comment:
but why? if i delete route 'post' for_tag won't pass (there is only definded get request)